inst/plumber/v1/endpoints.R

```diff
@@ -281,22 +276,22 @@ function(req) {
 #* @param welfare_type:[chr] Welfare Type. Options are "income" or "consumption"
 #* @param reporting_level:[chr] Reporting level. Options are "national", "urban", "rural".
 #* @param ppp:[dbl] Custom Purchase Power Parity (PPP) value.
-#* @param version:[chr] Data version. Defaults to most recent version. See api/v1/versions for all available versions
+#* @param data_version:[chr] date when the data was published in YYYYMMDD format
+#* @param ppp_version:[chr] ppp year to be used
 #* @param format:[chr] Response format. Options are "json", "csv", or "rds".
 #* @serializer switch
 function(req) {
   # Process request
-  # browser()
   params <- req$argsQuery
   params$lkup <- lkups$versions_paths[[params$version]]
   params$format <- NULL
   params$version <- NULL
 
   # Define default arguments
-  # if (is.null(params$country))
-  #   params$country <- "all"
-  # if (is.null(params$year))
-  #   params$year <- "all"
+  if (is.null(params$country))
+    params$country <- "all"
+  if (is.null(params$year))
+    params$year <- "all"
```

Comment:
I have uncommented these lines so the default `pip` API call works without the `country` and `year` argument.  I was curious why was this commented in the first place? Mostly, I am not aware of a certain scenario here.

Reply:
The default argument values are handled at the `pip()` function level, so I don't think these lines are necessary.

Reply:
Oh I see... This is needed for the parallel processing below. Mmm... I don't remember exactly why this has been commented out, but I think this is taken care of in one of the filters, so we should never get NULL values ofr country and year at this stage. To be checked.